server/util.go

```diff
@@ -65,6 +66,26 @@ func PersistentPreRunEFn(context *Context) func(*cobra.Command, []string) error
 	}
 }
 
+// If a new config is created, change some of the default tendermint settings
+func interceptLoadConfig() (conf *cfg.Config, err error) {
+	tmpConf := cfg.DefaultConfig()
+	viper.Unmarshal(tmpConf)
```

Comment:
What does this line do? We don't seem to read from `viper` anywhere else in this function?